include/event_manager.h

```diff
@@ -279,6 +202,28 @@ void *event_manager_alloc(size_t size);
 void event_manager_free(void *addr);
 
 
+/** Log event
+ *
+ * This helper macro simplifies event logging.
+ *
+ * @param eh  Pointer to the event heder.
+ * @param ... `printf`- like format string and variadic list of arguments corresponding to
+ *            the format string.
+ */
+#define EVENT_MANAGER_LOG(eh, ...) do {								\
```

Comment:
Discussed offline. The main problem is the fact that all of the events must align to the same logging convention (both from CAF and from user application). Aligning this may be difficult (if not impossible).

I think you can either remove this change from the PR or align all of the existing applications to use new macro (we cannot make it backwards compatible in the proposed way I think).